config/locales/en.yml

```diff
@@ -281,6 +281,8 @@ en:
       working: Working, jobs and pensions
       working_description: Includes holidays, finding a job and redundancy
     index:
+      ministerial_appointments_september_2021: "Ministerial appointments: September 2021"
+      ministerial_appointments_september_2021_text: "Latest updates as Prime Minister Boris Johnson makes government appointments."
       afghanistan: "Afghanistan: get support"
```

Comment:
Should this afghanistant text remain?

Reply:
Sure removed it and force pushed, we can leave a link to this PR if we return this stuff after the appointments proces is complete.

Reply:
[see here](https://github.com/alphagov/frontend/pull/2940/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7L284-L285)